Automattic/wp-calypso
# Keep early tests from failing due to conflicts when saving login info.

#### Changes proposed in this Pull Request

* This catches some of the errors when saving login info for re-use.  Often times when tests are running in parallel, several of the first tests will fail the first time. This is due to timing of creating the files to save login info. This PR attempts to keep the tests from failing when the happens.

#### Testing instructions

Ensure e2e tests pass in CI and that there aren't groups of failed tests at the beginning of the run.
